eng/versioning/version_client.txt

```diff
@@ -82,6 +82,7 @@ com.azure:azure-core-serializer-json-gson;1.1.18;1.2.0-beta.1
 com.azure:azure-core-serializer-json-jackson;1.2.19;1.3.0-beta.1
 com.azure:azure-core-test;1.10.0;1.11.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.26;1.0.0-beta.27
+com.azure:azure-core-metrics-opentelemetry;1.0.0-beta.1;1.0.0-beta.1
```

Comment:
nit: let's move this line below `azure-core-management` to maintain the lexicographical order